### PR TITLE
Feat/vehicle types table

### PR DESCRIPTION
**Pull request by Valentin271**
# Description

This PR adds the `vehicle_types` migrations and models.

I also documented a bit more the process of creating a new table and the tools required.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,4 +36,12 @@ Finally, this enables a `migrations` module which embed the migrations for later
 6. With *diesel* feature, it derives from `Queryable`. You can also specify the table name if needed
 7. The model file should also contains `struct` for a new model and an updating model
 
+
+You can quickly setup a database and run the migration with the following commands. This is actually required to generate the schema file.
```
Comment (kilian-goetz) [inline README.md:40]:
```suggestion
You can quickly setup a database and run the migration with the following commands. This is curently required to generate the schema file.
```

File: README.md [hunk 2]
```diff
@@ -14,6 +14,18 @@ There are types for a complete record, a new one or an updating one.
 `diesel` feature enables *derive* attributes on models for querying. It also enables the `schema` module for compile time query validation.  
 Finally, this enables a `migrations` module which embed the migrations for later run.
 
+## Tools
+
+This project needs the rust tooling, including cargo.  
+The diesel cli is needed, you can install it with the following command.
```
Comment (kilian-goetz) [inline README.md]:
```suggestion
The diesel CLI is needed, you can install it with the following command.
```

File: README.md [hunk 3]
```diff
@@ -14,6 +14,18 @@ There are types for a complete record, a new one or an updating one.
...
+
+```sh
+sudo apt install libmysqlclient-dev  # MySQL deps
+cargo install diesel_cli --no-default-features --features mysql
+```
+
+This project also needs a **MySQL** database, the easiest way to have one is with docker.
```
Comment (kilian-goetz) [inline README.md]:
```suggestion
This project also needs a **MySQL** database, the easiest way is using docker.
```

I think it's shortest and easiest to understand like so.

**Review by kilian-goetz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Valentin271) [reply README.md:40]: I really wanted to mean "in fact" and not "right now", 'actually' is the right word.

**Review by kilian-goetz — APPROVED** [review]

**Pull request merged**